src/neural_de.jl

```diff
@@ -443,6 +443,15 @@ struct NeuralODEMM{M,M2,P,RE,T,MM,A,K} <: NeuralDELayer
             typeof(tspan),typeof(mass_matrix),typeof(args),typeof(kwargs)}(
             model,constraints_model,p,re,tspan,mass_matrix,args,kwargs)
     end
+
+    function NeuralODEMM(model::TensorLayer,constraints_model,tspan,mass_matrix,args...;
```

Comment:
shouldn't the TensorLayer be used in a FastChain?

Reply:
That's true. The model that should be a TensorLayer.

Reply:
I don't think so. I think it should be the model itself.